--- /Scratch/john/tmp.vKHTg31290/share/pspp/data-editor.glade	2007-03-31 12:55:02.000000000 +0800
+++ src/ui/gui/data-editor.glade	2007-03-31 13:04:27.000000000 +0800
@@ -494,7 +494,7 @@
                     <property name="sensitive">False</property>
                     <property name="tooltip" translatable="yes">Variables</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-goto-variable</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -535,7 +535,7 @@
                     <property name="sensitive">False</property>
                     <property name="tooltip" translatable="yes">Insert Case</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-insert-case</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -547,7 +547,7 @@
                     <property name="sensitive">False</property>
                     <property name="tooltip" translatable="yes">Insert Variable</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-insert-variable</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -567,7 +567,7 @@
                     <property name="visible">True</property>
                     <property name="tooltip" translatable="yes">Split File</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-split-file</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -578,7 +578,7 @@
                     <property name="visible">True</property>
                     <property name="tooltip" translatable="yes">Weight Cases</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-weight-cases</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -590,7 +590,7 @@
                     <property name="sensitive">False</property>
                     <property name="tooltip" translatable="yes">Select Cases</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-select-cases</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
@@ -610,7 +610,7 @@
                     <property name="visible">True</property>
                     <property name="tooltip" translatable="yes">Value Labels</property>
                     <property name="use_underline">True</property>
-                    <property name="stock_id">gtk-missing-image</property>
+                    <property name="stock_id">pspp-value-labels</property>
                   </widget>
                   <packing>
                     <property name="expand">False</property>
